website_user_login_redirect_custom/models/res_config_settings.py

```diff
@@ -0,0 +1,73 @@
+# © 2025 Cetmix OU
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from urllib.parse import unquote, urlparse
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    website_login_redirect_enabled = fields.Boolean(
+        string="On Login/Signup Redirect",
+        config_parameter="website_user_login_redirect_custom.enabled",
+        help="If checked, users will be redirected to a custom URL after login/signup.",
+    )
+
+    website_login_redirect_url = fields.Char(
+        string="Custom URL",
+        config_parameter="website_user_login_redirect_custom.url",
+        help="Relative URL path where users will be redirected after login/signup. "
+        "Example: '/', '/shop', '/thank-you-for-joining-us'.",
+    )
+
+    @staticmethod
+    def is_valid_redirect_url(url) -> bool:
+        """Validate that URL is a safe relative path for redirects"""
+        if not url:
+            return False
+
+        url = url.strip()
+
+        # Must start with single '/', not '//' and not contain schemes
+        if not (url == "/" or (url.startswith("/") and not url.startswith("//"))):
+            return False
+
+        # Parse and ensure no scheme or netloc
+        parsed = urlparse(url)
+        if parsed.scheme or parsed.netloc:
+            return False
+
+        # Security checks with URL decoding to prevent encoding bypass
+        decoded_url = unquote(url)
+        if any(
```

Comment:
What's the reason behind checking the schema here? In line 40 you return False if there are any schemas in the URL.